.github/workflows/ci-dev-app.yml

```diff
@@ -63,11 +62,16 @@ jobs:
       - name: Pull Vercel Environment Information
         run: vercel pull --yes --environment=preview --cwd ../.. --token=${{ secrets.VERCEL_TOKEN }}
 
+        # NOTE (JL,2024-07-09): We create a `.env.local` file on the fly, to get the server-side Env Variables past
+        #   the `vercel build` invocation.
       - name: Build Project Artifacts
-        run: vercel build --cwd ../.. --token=${{ secrets.VERCEL_TOKEN }}
-        env:
+        run: |
+          cat >.env.local <<EOS
           API_BASE_URL: http://localhost:3001
           SUPABASE_SERVICE_ROLE_KEY: ${{ secrets.TEST_SUPABASE_SERVICE_ROLE_KEY }}
+          EOS
+          vercel build --cwd ../.. --token=${{ secrets.VERCEL_TOKEN }}
+        env:
```

Comment:
once we get the app build passing, suggest we try via github actions whether this passes or something to your local env.  i didn't need to do this to get act passing locally.

Reply:
I have done all my final testing in GitHub Actions, so they are passing. This is my issue with the hack, it wasn't needed before.